[BD-3687] Remove mention of 'automatic tracking' (#8554)
braze-inc/braze-docs: _docs/_user_guide/getting_started/users_segments.md

#### Standard attributes

> Standard attributes are predefined attributes that can be tracked by Braze after you integrate the Braze SDK into your app. These attributes are common pieces of user information that most apps would find useful, such as demographics and device data. Examples include:

Author: ```suggestion
Standard attributes are predefined attributes you can track with Braze after integrating the SDK into your app. They are common pieces of user information that most apps would find useful, such as demographics and device data. Examples include:
```
